index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-wrapper">
+      <header class="toolbar">
+        <div class="logo-place"><h3 class="my-logo hex">My Logo</h3></div>
+        <figure class="icons">
+          <img src="images/Icons/Union.png" alt="Icons" />
+        </figure>
```

Comment:
Kindly wrap your hamburger button in a `<nav>` tag 🙏. This is for semantic reasons as the hamburger button links to the mobile menu and is hence a navigation element